src/utils/pagination.ts

```diff
@@ -1,7 +1,7 @@
 import { PaginatedParameters } from '@dcl/schemas'
 
 export function getPage(limit: number, offset: number = 0) {
-  return Math.ceil(offset / limit) + 1
+  return limit ? Math.ceil(offset / limit) + 1 : 1
```

Comment:
We need to somehow define a maximum and default limit for the request to avoid issues with big queries. I'll take notes so we do it after the refactors.